reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/CreatePotentialIssuesReportRuleProvider.java

```diff
@@ -29,7 +28,7 @@
 /**
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
  */
-@RuleMetadata(phase = ReportGenerationPhase.class)
+@RuleMetadata(disabled = true, phase = ReportGenerationPhase.class)
```

Comment:
Rather than inventing a new way to do this for Java rules, just mark it @Vetoed. :)